content/news/buzz/november-2024-status-update/contents.lr

```diff
@@ -0,0 +1,56 @@
+title: November 2024 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+In November, BeeWare made more good progress on mobile packaging - however, that work has revealed a number of upstream tools that need modification before we can contribute wheels for mobile platforms.
+
+What we've done
+----------------
+
+* We attended `PyCon AU <https://2024.pycon.org.au>`__; video of the talk should be available on YouTube in the coming days. We also had a very productive sprint - 9 contributors earned their `challenge coin <https://beeware.org/contributing/challenge-coins/>`__, for contributions ranging from `corrections to the tutorial <https://github.com/beeware/beeware/pull/450>`__, to the `addition of a dark mode flag to Toga <https://github.com/beeware/toga/pull/2992>`__. We also gave out one Yak Herder coin, for someone who helps other people to get their challenge coin. A huge thank you to everyone that came along and participated!
```

Comment:
The video will likely go up over the weekend - I'll add a link if it happens before Monday.